src/Services/PatientService.php

```diff
@@ -14,6 +14,7 @@
 
 namespace OpenEMR\Services;
 
+require_once("{$GLOBALS['srcdir']}/patient.inc");
```

Comment:
when bring in cases like this (in an autoloaded class), generally use "hard-coded" path:
```
require_once(dirname(__FILE__) . "/../../library/patient.inc");
```
And to make it clear, generally make a line of spacing between this and the `use` entries

Reply:
Thanks! Will keep that in mind :)